lib/physical_communicator.hpp

```diff
@@ -13,7 +13,7 @@ class PhysicalCommunicator
   PhysicalCommunicator(LogicalAddress la) : subnetAddress(la) {}
 
   virtual ~PhysicalCommunicator() {}
-  virtual bool send(std::shared_ptr<SpaMessage> message) { return false; }
+  virtual bool sendMsg(std::shared_ptr<SpaMessage> message) { return false; }
```

Comment:
Are there any other classes that inherit from `PhysicalCommunicator` that need to be updated as well?